vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go

```diff
@@ -130,13 +166,56 @@ func Relabel(path string, fileLabel string, shared bool) error {
 		return nil
 	}
 
-	exclude_paths := map[string]bool{"/": true, "/usr": true, "/etc": true, "/tmp": true, "/home": true, "/run": true, "/var": true, "/root": true}
+	exclude_paths := map[string]bool{
```

Comment:
pls double-check if the new list will cause any changes in behavior that we should be aware of 🤗 (left the same question on https://github.com/moby/moby/pull/38918#pullrequestreview-217130793)